src/main/java/taxi/model/Driver.java

```diff
@@ -15,6 +17,29 @@ public Driver(String name, String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public Driver(String name, String licenseNumber, String login, String password) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+        this.login = login;
+        this.password = password;
+    }
```

Comment:
```suggestion
    public Driver(String name, String licenseNumber, String login, String password) {
        this(name, licenseNumber);
        this.login = login;
        this.password = password;
    }
```
Remember about this() when you create a lot of constructors